<?xml version="1.0"?>
<launch>
    <node name="soundplay_node" pkg="sound_play" type="soundplay_node.py" output="screen"/>
    <node name="audio_status_reporter" pkg="humane" type="audio_status_reporter.py" output="screen">
        <param name="mp3dir" value="$(find humane)/audio"/>
    </node>
    
    <!--include file="$(find novelti)/test/launch/test_keyboard_navigation.launch"/-->
</launch>